src/Pages/Seokmoon/Login/Login.js

```diff
@@ -1,7 +1,132 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
-function Login() {
-  return <div>Seokmoon Login</div>;
-}
+const Login = () => {
+  // const [userId, setId] = useState('');
+  // const [userPw, setPw] = useState('');
+  const [inputValues, setInputValues] = useState({
+    email: '',
+    password: '',
+  });
+  const [loginBtn, setButton] = useState(true);
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    setInputValues({ ...inputValues, [name]: value });
+  };
+
+  // const saveInputId = e => {
+  //   setId(e.target.value);
+  // };
+
+  // const saveInputPw = e => {
+  //   setPw(e.target.value);
+  // };
+
+  const navigate = useNavigate();
+
+  // const goToMain = () => {
+  //   navigate('/mains');
+  // };
+
+  const clickButton = e => {
+    setButton(e.target.value);
+    inputValues.email.includes('@') > 0 && inputValues.password.length >= 8
+      ? setButton(false)
+      : setButton(true);
+  };
```

Comment:
```suggestion
  const isValid = inputValues.email.includes('@') > 0 && inputValues.password.length >= 8;
```
- 이렇게 한다면 동적으로 관리가 되고 state는 필요 없을 것 같습니다.